dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumBackgroundScope.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.internal.domain.scope
+
+import com.datadog.android.core.internal.net.FirstPartyHostDetector
+import com.datadog.android.rum.internal.domain.Time
+
+internal class RumBackgroundScope(
+    parentScope: RumScope,
+    eventTime: Time,
+    initialAttributes: Map<String, Any?>,
+    firstPartyHostDetector: FirstPartyHostDetector
+) : RumViewScope(
```

Comment:
Not sure if background view is specialising the view scope. It's IMO too early to make this assumption and it might be safer to just start with allocating distinct instance of regular `RumViewScope` with passing `"com/datadog/background/view"` and `"BackgroundView"` in its constructor. We may shortly realise that background views are "slightly" different and this inheritance will start leaking. It might be easier to embrace it if "background" is determined by scope's property not identity. WDYT?

Reply:
You're right, actually the reason why I added this subclass what to add special logic inside (if needed). Right now I realize is not needed anymore and I can just start a simple `RumViewScope` with a dedicated name and url